doc/spec/source/containers/containers.rst

```diff
@@ -15,4 +15,10 @@ processes and can be accessed by any process in the team.
    :maxdepth: 1
 
    mhp_distributed_vector
+   mhp_distributed_dense_matrix
+
+   shp_vector
+   shp_device_vector
```

Comment:
```suggestion
```

I believe this is internal and should not be documented.

Reply:
If so, shouldn't it be in __detail? @BenBrock , can you give any hints on what classes to include in the documentation?